### PR TITLE
Update readme

### PR DESCRIPTION
**Pull request by MichaelYoung1981**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,165 +1,55 @@
...
+| CacheConfiguration:DefaultCache    | empty                                    |
+| CacheConfiguration:CacheConnection | empty                                    |
+| ResourceEnvironmentName            | LOCAL                                    |
+| ApprenticeshipsOuterApi:BaseUrl    | https://localhost:7101/                  |
 
-## 🐛 Known Issues
+## 🔗 External Dependencies
 
-_Add any known issues with the project_
+The UI should only be dependant on the Apprenticeships Outer API.  If you want to run the application locally you will need to make sure the Outer API is running locally or the configuration is pointing to a deployed instance.
```
Comment (tom-gough) [inline README.md:48]: Mention inner api here too?

**Review by tom-gough — CHANGES_REQUESTED** [review]

**Review by MikeV-TalentConsulting — APPROVED** [review]

**Review by tom-gough — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -1,165 +1,65 @@
...
 
-# _Project Name_
+# Apprenticeships Web
 
 <img src="https://avatars.githubusercontent.com/u/9841374?s=200&v=4" align="right" alt="UK Government logo">
 
-_Update these badges with the correct information for this project. These give the status of the project at a glance and also sign-post developers to the appropriate resources they will need to get up and running_
-
-[![Build Status](https://dev.azure.com/sfa-gov-uk/Digital%20Apprenticeship%20Service/_apis/build/status/_projectname_?branchName=master)](https://dev.azure.com/sfa-gov-uk/Digital%20Apprenticeship%20Service/_build/latest?definitionId=_projectid_&branchName=master)
+[![Build Status](https://dev.azure.com/sfa-gov-uk/Digital%20Apprenticeship%20Service/_apis/build/status/das-apprenticeships-web?branchName=master)](https://dev.azure.com/sfa-gov-uk/Digital%20Apprenticeship%20Service/_build/latest?definitionId=3497&branchFilter=112959%2C112959%2C112959)
```
Comment (tom-gough) [inline README.md:7]: This is a specific build - should be https://dev.azure.com/sfa-gov-uk/Digital%20Apprenticeship%20Service/_build?definitionId=3497&_a=summary

File: README.md [hunk 3]
```diff
@@ -1,165 +1,65 @@
...
-    "DatabaseName": "QueueExplorer",
-    "CollectionName": "Session",
-    "Throughput": 400,
-    "DefaultCosmosOperationTimeout": 55,
-    "DefaultCosmosInterimRequestTimeout": 2
-  }
-}
-```
+* A code editor that supports .Net6
+* Azure Storage Emulator (Azureite)
```
Comment (tom-gough) [inline README.md:39]: typo in Azurite

**Review by tom-gough — DISMISSED** [review]
Approved but added 2 minor comments

**Review by tom-gough — APPROVED** [review]

**Pull request merged**